src/components/TeaserCarousel/Row.js

```diff
@@ -146,9 +146,8 @@ const Row = ({ children }) => {
           {...styles.arrowBg}
           {...colorScheme.set('backgroundColor', context.bgColor)}
         />
-        <ChevronLeft
+        <ChevronLeftIcon
           size={50}
-          {...styles.arrowIcon}
```

Comment:
Accidental removal? if not why is the `position: 'relative'` no longer necessary? and then also remove from the style object above.